tests/runfiles/linux.run

```diff
@@ -18,29 +18,35 @@ timeout = 600
 post_user = root
 post = cleanup
 outputdir = /var/tmp/test_results
+tags = []
```

Comment:
Why is a tags field  needed here at all?

Reply:
Refreshed. Defaulted to having the tag `functional` for any test group without tags specified.